src/main/assets/js/opening-hours.ts

```diff
@@ -0,0 +1,88 @@
+import $ from 'jquery';
+
+const formId = '#openingTimesForm';
+const tabId = '#openingTimesTab';
+const newOpeningTimeHeadingId = '#newOpeningHoursHeading';
+const openingTimesContentId = '#openingTimesContent';
+const deleteBtnClass = 'deleteOpeningTime';
+const addOpeningTimesBtnName = 'addOpeningTime';
+const typeSelectName = 'type_id';
```

Comment:
Are these methods private? 

Also, is there a naming convention for variables that are local to the scope of the file in javascript? 

For example (Global/Class Constants): 
java: TYPE_SELECT_NAME = 'type_id' < if at the top 
python: typically the same as java, but with _ at the start if private 

happy to take your steer on this one.

Reply:
I've refactored this into a proper typescript class and made these variables private.